javascript/skarmbot.js

```diff
@@ -423,11 +423,12 @@ class Bot {
             default:
                 break;
         }
+        
 
         // gets all the guilds that this user is in, and the open polls in each one
-        let openPolls = userData.getGuilds()                    // get all guilds the user is in
-            .flatMap(g => g.anonPoll.polls.filter(p => p.open)  // filter down the polls in each of those guilds to just the ones that are open to new submissions
-                .map((p) => { return { guild: g, poll: p } })); // return pairs of guild and poll to be able to print out the source guild of each poll
```

Comment:
This was done to eliminate a circular dependency between Users.js and Guilds.js

Reply:
The circular dependency was preventing Users from successfully initializing to a usable state.